Utils/hdinsight-node-common/src/com/microsoft/azure/hdinsight/sdk/common/SharedKeyHttpObservable.java

```diff
@@ -97,6 +95,12 @@ public SharedKeyHttpObservable setContentType(@NotNull String type) {
         return super.executeReqAndCheckStatus(req, validStatueCode, pairs);
     }
 
+    @Override
+    public <T> Observable<T> get(String uri, List<NameValuePair> parameters, List<Header> addOrReplaceHeaders, Class<T> clazz) {
```

Comment:
What's the method used for?

Reply:
to list files in the adlsgen2 fs operation

---
In reply to: [283615602](https://github.com/microsoft/azure-tools-for-java/pull/3124#discussion_r283615602) [](ancestors = 283615602)

Reply:
I suppose the authorization should be a part of default header.

Reply:
for get api ,we need to set the auth according to the parameters

---
In reply to: [286317545](https://github.com/microsoft/azure-tools-for-java/pull/3124#discussion_r286317545) [](ancestors = 286317545)

Reply:
it is in default header,but we need to set auth according to the parameters

---
In reply to: [286317545](https://github.com/microsoft/azure-tools-for-java/pull/3124#discussion_r286317545) [](ancestors = 286317545)

Reply:
Need a discuss.